.gitignore

```diff
@@ -9,6 +9,9 @@ cdk.out
 !.vscode/settings.json.example
 !.vscode/extensions.json
 
+# JetBrains IDEs
+.idea/
```

Comment:
Curious - what is this?

Reply:
This is analogous to the `.vscode/` folder, but used by the JetBrain IDEs such as PyCharm and WebStorm. For those of us (maybe just me here?) who use one of these IDEs, these files will be kicking around in the Git diffs waiting to be accidentally committed, without adding them to `.gitignore`.